modules/ROOT/pages/supported-cluster-config.adoc

```diff
@@ -73,7 +73,7 @@ For both 4-node and 7-node configurations, the following apply:
 
 * A single API implementation with one policy counts as 0.5 APIs.
 * A single clustered proxy with up to 5 policies counts as 1.
-* All runtimes are polling every 60 seconds.
+* All Mule runtimes are polling every 60 seconds.
```

Comment:
As per the term guide, instead of **Mule runtimes** we should use either _Mule runtime engine_ or _Mule instances_, I'd suggest the latter for this case

Reply:
Thanks. Change globally.